### ISSUE TITLE
Services aren't editable due broken URL validation

### ISSUE THREAD

**Issue by codekandis** [ramboxapp/community-edition#2476]
If I want to edit a service, e. g. just checking "align to right", it won't save the changes while it shows the URL is invalid.

As far as I know it didn't happen in 0.7.0 because I edited some services then. So I assume it broke in 0.7.1 which I updated yesterday. My recent update to 0.7.2 didn't fixed it.

**Comment by fvulich** [comment]
Could you tell us which URL it's throwing the error?

**Comment by codekandis** [comment]
@fvulich Due to your question I tried to determine it more specifically now and can prove it happens in custom services only. The URL doesn't matter.

What I disovered now is, that while editing, the URL will have been stripped one of the double slashes after the scheme. I didn't catch that earlier.

If I insert the second slash while editing and save the config and edit it again the slash will be stripped again. This definitely causes the invalidation error.
